### PR TITLE
Adds confetti!

### PR DESCRIPTION
**Pull request by ComputasAlex**
Fun crazy I/O confetti when new highscore!

Multiplayer: When winning

### CODE REVIEW

**Comment by ComputasAlex** [comment]
https://github.com/user-attachments/assets/d9ccd5d2-b816-4d2c-9fa6-ecfa770111d0

File: src/app/game/game-result/game-result.component.ts [hunk 1]
```diff
@@ -123,6 +131,28 @@ export class GameResultComponent implements OnInit, OnDestroy {
     this.startAnimation();
     this.translationService.loadTranslations(this.translationService.getCurrentLang()).subscribe();
   }
+  blowConfetti(): void {
+    const confettiEndTime = Date.now() + this.confettiDuration;
```
Comment (vkallerud) [inline src/app/game/game-result/game-result.component.ts:151]: are you referring to `confettiEndTime` any other places in the code? You have also declared this as a variable on line 54.

**Review by vkallerud — COMMENTED** [review]
Will re-review for approval once `develop` has been merged into this branch!☺️ But looks great! Fun confetti 🎉

**Review by andreasrberg — APPROVED** [review]
Love it!

**Pull request merged**